src/config.xml

```diff
@@ -565,6 +565,17 @@ Go to the <a href="commands.html">next</a> section or return to the
 ]]>
       </docs>
     </option>
+    <option type='enum' id='SNIPPET_TRIMLEFT' defval='NO'>
+      <docs>
+<![CDATA[
+ Inserted snippet will not trimmed by default.
+ 
+ Setting \c SNIPPET_TRIM_LEFT to YES will remove lines common starting spaces/tab.
+]]>
+      </docs>      
```

Comment:
- ... will not trimmed .... -> ... will not be trimmed ....
- to YES -> to \c YES
- there is an inconsistency in the name `SNIPPET_TRIMLEF` and later `SNIPPET_TRIM_LEFT`
- the line starting with "Setting" could be made a bit more explicit and also refer to the `TAB_SIZE` setting, maybe something like:
  ```
  Setting \c SNIPPET_TRIMLEFT to \c YES will remove from the lines in the snippet the starting spacing that is common to all the lines (for `<tabs>` also the setting of  \ref cfg_tab_size "TAB_SIZE" is taken into consideration.
  ```
- why is an `enum` used and not a `bool` as type?

Reply:
regarding the last remark, regarding `enum` <-> `bool` this is probably because of the comment that it still in the code:
>   // to be used in case of implement SNIPPET_TRIMLEFT = TABCONVERT
>   // auto tabSize = Config_getInt(TAB_SIZE); 

so looks like some work in progress.